app/gather/assets/apps/utils/avro-utils.spec.jsx

```diff
@@ -102,12 +102,12 @@ describe('AVRO utils', () => {
         ]
       }
       const expected = {
-        labels: {first: 'The first'},
+        labels: { first: 'The first' },
         paths: ['first']
       }
-      assert.deepEqual(extractPathDocs(schema), expected)
+      assert.deepStrictEqual(extractPathDocs(schema), expected)
```

Comment:
Any specific reason for making all these strict, or are you just being extra careful?

Reply:
Nop, `equal`/`deepEqual` were deprecated long time ago and node suggests or enforces to use the `strict` method.

https://nodejs.org/api/assert.html#assert_assert_equal_actual_expected_message

Reply:
Ah, should've read your original comment ...